packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -1616,6 +1620,81 @@ describe('ReactSuspenseWithNoopRenderer', () => {
     expect(ReactNoop.getChildren()).toEqual([span('Loading...')]);
   });
 
+  it('warns when a low pri update suspends inside a high pri call for functional components', async () => {
```

Comment:
Terminology nit: Replace "call" with "update" since that's the term we usually use for this concept